cirro/cli/controller.py

```diff
@@ -142,6 +143,15 @@ def run_download(input_params: DownloadArguments, interactive=False):
         datasets = cirro.datasets.list(project_id)
         dataset_id = get_id_from_name(datasets, input_params['dataset'])
 
+        if (input_params['files']):
+            all_files = cirro.datasets.get_file_listing(project_id, dataset_id)
+            files_to_download = []
+            for filepath in input_params['files'].split(','):
+                if not filepath.startswith('data/'):
+                    filepath = os.path.join('data/', filepath)
+                file = [f for f in all_files if f.relative_path == filepath][0]
```

Comment:
This will give an `IndexError` to the user if the file wasn't found. Can you use the `next` pattern here to check to see if the file is valid and if not throw a better message.
```
        matched_process = next((p for p in self.list() if p.name == name), None)
        if not matched_process:
            return None
```